lavalink/websocket.py

```diff
@@ -19,6 +19,10 @@ def __init__(self, node, host: str, port: int, password: str):
         self._host = host
         self._port = port
         self._password = password
+        self._resume_key = resume_key
+        self._resume_timeout = resume_timeout
+
+        self._resuming_configured = False
```

Comment:
I feel bad about this bool, is there a better way of accomplishing this ?
 Such as setting `_resume_key` to `None` or something

Reply:
Resume key is used later on *when* resuming -- it's sent through WebSocket headers

Reply:
Huh, right